From 7c65a3f2231ea3fc17c7e0a90ebf98a47dfb42b3 Mon Sep 17 00:00:00 2001
From: Sven Schwermer <sven@svenschwermer.de>
Date: Sun, 21 Jul 2019 18:35:52 +0200
Subject: [PATCH 2/3] asoc: wm8731: Always enable oscillator for DAC

---
 sound/soc/codecs/wm8731.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/codecs/wm8731.c b/sound/soc/codecs/wm8731.c
index 6fd1bef848ed..4ef33c5a7afc 100644
--- a/sound/soc/codecs/wm8731.c
+++ b/sound/soc/codecs/wm8731.c
@@ -218,7 +218,7 @@ static int wm8731_check_osc(struct snd_soc_dapm_widget *source,
 }
 
 static const struct snd_soc_dapm_route wm8731_intercon[] = {
-	{"DAC", NULL, "OSC", wm8731_check_osc},
+	{"DAC", NULL, "OSC"},
 	{"ADC", NULL, "OSC", wm8731_check_osc},
 	{"DAC", NULL, "ACTIVE"},
 	{"ADC", NULL, "ACTIVE"},
-- 
2.24.1

